code/controllers/subsystem/blackmarket.dm

```diff
@@ -0,0 +1,162 @@
+SUBSYSTEM_DEF(blackmarket)
+	name		 = "Blackmarket"
+	flags		 = SS_BACKGROUND
+	init_order	 = INIT_ORDER_DEFAULT
+
+	// I don't know why you would need this but it can be fun.
+	var/supplypod_type = /obj/structure/closet/supplypod
```

Comment:
```suggestion
	var/supplypod_type = /obj/structure/closet/supplypod
	temp_pod.setStyle(STYLE_SYNDICATE)
```

Reply:
you should do this to make the pod look more suspicious
You can see a lot more settings like style/sound/damage/traveltime in centcom_podlauncher.dm

Reply:
The black market isn't syndicate themed though.

Reply:
could make it the airdrop crate